delve.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: delve
+  version: 1.20.1
+  epoch: 0
+  description: Delve is a debugger for the Go programming language.
+  target-architecture:
+    - all
+  copyright:
+    - license: MIT
+      paths:
+        - "*"
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - build-base
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/go-delve/delve/archive/refs/tags/v${{package.version}}.tar.gz
+      expected-sha256: a10aa97d3f6b6219877a73dd305d511442ad0caab740de76fc005796a480de93
+  - runs: |
+      make build
+      mv dlv ${{targets.destdir}}/
```

Comment:
```
mkdir -p ${{targets.destdir}}/usr/bin/
mv dlv ${{targets.destdir}}/usr/bin/
```
?

Reply:
Done! I forget that `${{targets.destdir}}` isn't just `/usr/bin` 🤦